jfr-daemon/src/main/java/com/newrelic/jfr/daemon/DaemonConfig.java

```diff
@@ -94,7 +94,7 @@ public boolean useSharedFilesystem() {
     return useSharedFilesystem;
   }
 
-  public boolean isUseLicenseKey() {
+  public boolean useLicenseKey() {
```

Comment:
I believe that one test in the java agent uses `isUseLicenseKey` so that will need to be updated whenever a version of the jfr-daemon artifact with this change is published and pulled into the agent.